src/settings.rs

```diff
@@ -9,13 +9,21 @@ use std::path::PathBuf;
 type CommitsMetadataSettings = HashMap<String, CommitConfig>;
 pub(crate) type AuthorSettings = Vec<AuthorSetting>;
 
+#[derive(Copy, Clone)]
+pub(crate) enum HookType {
+    PreBump,
+    PostBump,
+}
+
 #[derive(Debug, Deserialize, Serialize)]
 pub(crate) struct Settings {
     pub changelog_path: Option<PathBuf>,
     pub github: Option<String>,
     #[serde(default)]
     pub hooks: Vec<String>,
```

Comment:
I think you can rename this to `pre_bump_hook` and create our first breaking change commit !

Reply:
Don't forget to update `cog.toml` and docs accordingly

Reply:
Done, also updated the bump test